x/smart/stream_dialer.go

```diff
@@ -94,10 +95,16 @@ type dnsEntryConfig struct {
 	TCP    *tcpEntryConfig   `yaml:"tcp,omitempty"`
 }
 
+type fallbackEntryConfig struct {
+	ConfigURL 	string 					`yaml:"configUrl,omitempty"`
+	// Don't verify the psiphon config format here, just pass it forward
+	Psiphon 	map[string]interface{} 	`yaml:"psiphon,omitempty"`
+}
+
 type configConfig struct {
-	DNS      []dnsEntryConfig `yaml:"dns,omitempty"`
-	TLS      []string         `yaml:"tls,omitempty"`
-	FALLBACK []string         `yaml:"fallback,omitempty"`
+	DNS      []dnsEntryConfig 		`yaml:"dns,omitempty"`
+	TLS      []string         		`yaml:"tls,omitempty"`
+	Fallback []fallbackEntryConfig 	`yaml:"fallback,omitempty"`
```

Comment:
How do we make sure that old clients don't crash on new configs?
An old client should ignore the fallback list or strategies it doesn't yet know about.

Reply:
I thought the point was to fit a few things into fallback as short term stopgaps. But that this would eventually be replaced by a different config that will be developed in the outline client?